src/controller/tests/data_model/TestRead.cpp

```diff
@@ -1436,6 +1436,12 @@ void TestReadInteraction::TestReadAttributeTimeout(nlTestSuite * apSuite, void *
     NL_TEST_ASSERT(apSuite, ctx.GetExchangeManager().GetNumActiveExchanges() == 0);
 }
 
+#if 0
```

Comment:
If we were using sufficient callback control over things here, we could have it resub without redoing CASE, no?

Reply:
We could, but I didn't feel inclined to do surgery on the the APIs in `ReadInteraction.h` to expose that level of control just for the sake of testing.

Reply:
We could do the testing with ReadClient directly...I could add that.

Reply:
I added both a C++ based unit-test that validates non-CASE based re-sub, as well as a Python based re-sub test.